daemon/pod/decommission.go

```diff
@@ -280,31 +296,6 @@ func (p *XPod) RemoveContainer(id string) error {
 	p.factory.registry.ReleaseContainer(id, c.SpecName())
 	delete(p.containers, id)
 
-	removedVols := make([]string, 0, len(cvols))
-	for _, cv := range cvols {
-		if v, ok := p.volumes[cv.Name]; ok {
-			removed, err := v.tryRemoveFromSandbox()
-			if err != nil {
-				v.Log(ERROR, "failed to unplug vol: %v", err)
-				continue
-			}
-			if !removed {
-				v.Log(DEBUG, "volume did not unplug because it is in use")
-				continue
-			}
-			v.Log(DEBUG, "volume unplugged")
-			removedVols = append(removedVols, cv.Name)
-		}
-	}
-
-	//remove volumes from daemondb
```

Comment:
@feiskyer intendedly, I just notice remove container shouldn't remove related volume from pod(at least there is no such logic here), so this should not store to db.